src/schemathesis/cli/__init__.py

```diff
@@ -640,7 +642,8 @@ def run(
                 base_url = base_url or test_run.config.base_url
         except requests.HTTPError as exc:
             handle_service_error(exc)
-
+    if report and client is None:
+        client = service.ServiceClient(base_url=schemathesis_io_url, token=None)
```

Comment:
Is it needed at all? there is no token, so it could be just a single `requests` call with the proper User-Agent.